### DISCUSSION TITLE
system config to run this smoothly?

### DISCUSSION THREAD

**Discussion by ashishnex007** [hacksider/Deep-Live-Cam#111] [Q&A]
How much RAM , VRAM needed and GPU to run this?

**Comment by hacksider** [comment]
Recommended is
Ram: 16gb
Cpu: 8700k or higher
Gpu: 8gb vram or higher (nvidia)

That will be smooth enough to process 12fps or higher.

**Reply by Albaricoque695** [reply]
Is the CPU a big deal?

**Reply by hacksider** [reply]
I don't know what kind of answer you are expecting on this after we give the reccomended hardware 😅

**Reply by Albaricoque695** [reply]
> I don't know what kind of answer you are expecting on this after we give the reccomended hardware 😅

What about the OS?

**Reply by hacksider** [reply]
Windows is the most optimal I can say but it will run on linux as well. But yeah, it all ends up with the hardware.

**Comment by JuanmaWL** [comment]
Is there any way to make it run smoothly with the enhanced face (with CUDA or other system)? That one runs at 3 fps hahaha.
I have a 13400F + 4070Ti + 32 GB of RAM, and only the normal version runs smoothly without face enhance.

**Reply by SabMehdi** [reply]
hello, i also have a high end system config, but still getting very low fps when enabling face enhancer, did you figure out a solution?

**Reply by hacksider** [reply]
Maximum we guarantee as of this moment is 20fps. Reason for that is the post processing that is happening before the rendering.

**Reply by firovskq** [reply]
> Maximum we guarantee as of this moment is 20fps. Reason for that is the post processing that is happening before the rendering.

any update when its gonna be smoother?

**Comment by tathienthanh** [comment]
Hi bro! I can't install torch, it say "don't match your enviroment". Can u show me how to fix this error?
![image](https://github.com/user-attachments/assets/ffdb3600-c34d-4da7-b69b-159851d7217f)

**Reply by ashishnex007** [reply]
your architecture doesn't match with the command I guess, create a new coda env and then check your arch and install cuda for your specific computer. Always create a new conda env before installing a new project, especially like these ones.

**Reply by ashishnex007** [reply]
follow this tut - https://www.youtube.com/watch?v=MICScLVM1x8
